src/main/resources/db/migration/V4__drop_planlagt_melding_index.sql

```diff
@@ -0,0 +1,2 @@
+DROP INDEX CONCURRENTLY planlagt_melding_fnrsd_idx;
```

Comment:
Disse indexene skulle aldri vært unike, så jeg dropper og gjenoppretter dem. Trekker ut begge scriptene i en egen PR, men beholder dem her enn så lenge for å få testet helheten.